file/writable_file_writer.h

```diff
@@ -191,8 +193,12 @@ class WritableFileWriter {
         checksum_finalized_(false),
         perform_data_verification_(perform_data_verification),
         buffered_data_crc32c_checksum_(0),
-        buffered_data_with_checksum_(buffered_data_with_checksum),
-        temperature_(options.temperature) {
+        buffered_data_with_checksum_(buffered_data_with_checksum)
+#ifndef ROCKSDB_LITE
```

Comment:
It is a little cleaner (code appearance-wise) if you just set temperature_ in the constructor body conditionally rather than as an argument initializer.  Another option would be to move the temperature_ to not be the last item to be initialized.

It is a minor nit, but will make code maintenance simpler going forward if new arguments/options are added.

Reply:
Good point- will fix it

Reply:
Fixed